src/Main.java

```diff
@@ -1,5 +1,49 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("задача 1");
+        int[] money = {1000, 1200, 1150, 2500, 2700};
+        int sum = 0;
+        for (int index = 0; index < money.length; index++) {
+            sum = sum + money[index];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+
+        System.out.print(" ");
+        System.out.println("задача 2");
+        int[] money2 = {200, 250, 310, 400, 150};
+        int sumMax1 = 0;
+        int sumMin = money2[4];
```

Comment:
Обычно приравнивают к первому элементу массива
```suggestion
        int sumMin = money2[0];
```